lib/dataSource-core.js

```diff
@@ -354,8 +363,10 @@ kettle.dataSource.CouchDB.read = function (that, resp) {
  * `that.options.rules.writePayload`. By default, the entirety of the model is wrapped in a `value` element to avoid
  * collisions with top-level CouchDB variables such as `_id` and `_rev`.
  *
- * @param {model} The data to be stored.
- * @param {options} The `dataSource` options (see above).
+ * @param {Component} that - The dataSource component, used to read the payload write transform option
+ * @param {Object} model - The data to be stored.
+ * @param {Object} options - The dataSource's request options (see above).
+ * @return {Promise} A promise which resolves to the transformed, written payload
```

Comment:
There are many examples of period-less return descriptions, moreso than parameters.